jupyter-book/_toc.yml

```diff
@@ -102,6 +102,7 @@ parts:
     - file: python_scripts/linear_regression_non_linear_link
     - file: python_scripts/linear_models_ex_02
     - file: python_scripts/linear_models_sol_02
+    - file: python_scripts/linear_models_feature_engineering_classification.py
     - file: python_scripts/logistic_regression_non_linear
```

Comment:
This new notebook makes the logistic_regression_non_linear notebook obsolete, shall we remove it?

Reply:
I wanted to convert it into an exercise (about the relation ship between nystrom + linear SVC on one hand and SVC with kernel on the other hand) but I plan to do that in a follow-up PR.